client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get, find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { updateSettings } from 'state/jetpack/settings/actions';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { getPluginsForSite } from 'state/plugins/premium/selectors';
+
+const SpamFilteringSettings = ( {
+	akismetActive,
+	akismetKey,
+	fields,
+	isRequestingSettings,
+	isSavingSettings,
+	onChangeField,
+	siteSlug,
+	translate
+} ) => {
+	const header = akismetActive
+		? translate( 'Your site is protected from spam' )
+		: (
+			<div>
+				<div>
+					{ translate( 'Your site is not protected from spam.' ) }
+				</div>
+				<FormSettingExplanation>
+						{
+							translate( '{{link}} Install & activate Akismet on your site{{/link}}.', {
+								components: {
+									link: <a
+										href={ `/plugins/${ siteSlug }` } />
+								}
+							} )
+						}
+				</FormSettingExplanation>
+			</div>
+		);
+
+	return (
+		<div>
+			<FoldableCard header={ header }>
+				<div className="spam-filtering-settings__module-settings">
+						<div className="spam-filtering-settings__info-link-container site-settings__info-link-container">
+							<InfoPopover >
+								<ExternalLink target="_blank" icon href={ 'https://jetpack.com/features/security/spam-filtering/' } >
+									{ translate( 'Learn more about spam filtering.' ) }
+								</ExternalLink>
+							</InfoPopover>
+						</div>
+					<FormFieldset>
+						<FormLabel>{ translate( 'Your API Key ' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **42** times:
`translate( 'API Key' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/akismet-card-for-jetpack-sites-in-settings-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).